### DISCUSSION TITLE
Adding announcementDuration to Firebase-Ace package

### DISCUSSION THREAD

**Discussion by lakbychance** [0xTheProDev/Operational-Transformation#82] [General]
Hey @Progyan1997 , now that the remote tooltip implementation is done, the `FireAceEditor `constructor can accept `announcementDuration` as in `FireMonacoEditor`.  So the change also has to go in `@otjs/firebase-ace` package. Plus README also has to be updated accordingly for both `@otjs/ace` and `@otjs/firebase-ace`. Can you raise an issue for the same ? TBH, I am not a fan of bugging the maintainers every time when raising an issue 🤔 .

**Comment by 0xTheProDev** [comment]
My bad, should have made this change as part of #76. Added #83
